platform/security/idp/security-idp-plugin/security-idp-plugin-namequalifier/pom.xml

```diff
@@ -0,0 +1,114 @@
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>ddf.security.idp</groupId>
+    <artifactId>security-idp-plugin</artifactId>
+    <version>2.12.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>security-idp-plugin-namequalifier</artifactId>
+  <name>DDF :: Security :: IdP :: Plugin :: Name Qualifier</name>
+  <packaging>bundle</packaging>
+
+  <dependencies>
+    <dependency>
+      <groupId>ddf.security.idp</groupId>
+      <artifactId>security-idp-plugin-api</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>ddf.security</groupId>
+      <artifactId>ddf-security-common</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>ddf.platform.util</groupId>
+      <artifactId>platform-util</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>ddf.platform.util</groupId>
+      <artifactId>platform-util-unavailableurls</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+
+    <!--Spock dependencies-->
+    <dependency>
+      <groupId>ddf.lib</groupId>
+      <artifactId>spock-shaded</artifactId>
+      <version>${project.version}</version>
+      <scope>test</scope>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-bundle-plugin</artifactId>
+        <configuration>
+          <instructions>
+            <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+            <Embed-Dependency>
+              ddf-security-common,
+              platform-util,
+              platform-util-unavailableurls
+            </Embed-Dependency>
+          </instructions>
+        </configuration>
+      </plugin>
+      <plugin>
+        <groupId>org.jacoco</groupId>
+        <artifactId>jacoco-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>default-check</id>
+            <goals>
+              <goal>check</goal>
+            </goals>
+            <configuration>
+              <haltOnFailure>true</haltOnFailure>
+              <rules>
+                <rule>
+                  <element>BUNDLE</element>
+                  <limits>
+                    <limit>
+                      <counter>INSTRUCTION</counter>
+                      <value>COVEREDRATIO</value>
+                      <minimum>0.97</minimum>
+                    </limit>
+                    <limit>
+                      <counter>BRANCH</counter>
+                      <value>COVEREDRATIO</value>
+                      <minimum>0.72</minimum>
+                    </limit>
+                    <limit>
+                      <counter>COMPLEXITY</counter>
+                      <value>COVEREDRATIO</value>
+                      <minimum>0.87</minimum>
```

Comment:
❓ Did you leave a .03 buffer on each of these numbers? Sometimes the numbers that jacoco calculates are slightly different on different OS's.